interfaces/PA-App/src/app/shared/dialogue-turn/dialogue-turn.component.html

```diff
@@ -14,9 +14,16 @@
     <ion-avatar *ngIf="isSelf" style="background:#2f4bfd; display:flex;">
       <ion-icon name="person" color="light" size="large" style="margin:auto;"></ion-icon>
     </ion-avatar>
-    <ion-avatar *ngIf="!isSystem && !isSelf" style="color:black; background:white; display:flex;">
-      <b style="margin:auto">{{ actor }}</b>
-    </ion-avatar>
+    <div *ngIf="!isSystem && !isSelf && actor === 'NGO A'; else noLogo">
```

Comment:
When we explicitly check in the template for the string-value of `actor`, we can do anything we want inside this `ngIf`-block, so also use any filename we want.

Also hard-coding the filename is better then using the string mashup of line 19.

Reply:
Also, this error is thrown when you try to build the interface with the `--prod` flag.

```
ERROR in src/app/shared/dialogue-turn/dialogue-turn.component.html(17,10): : This condition will always return 'false' since the types 'Actor' and '"NGO A"' have no overlap.
```

So I think you need to include the ngo's we want into the `Actor`-enum in `dialogue-turn.component.ts`

Reply:
This error is happening on the test-vm on deployment.